conditional/templates/intro_evals.html

```diff
@@ -169,8 +182,90 @@ <h4>Other Comments</h4>
 
 </div>
 {% endfor %}
+        </div>
```

Comment:
I'm not really sure which block this is supposed to align to? (I want to say `<div class="row mobile-hide">`)